src/messageComposer/middleware/textComposer/commands.ts

```diff
@@ -143,29 +148,49 @@ export const createCommandsMiddleware = (
           return next(newState);
         }
 
+        const inputText = triggerWithToken?.toLowerCase().slice(1);
+        const matchedCommand = commands.find((command) => {
```

Comment:
why are you not using the search source?

Reply:
I will check. This can be used.

Reply:
Question:

Is it fine to make the middleware async and get commands from `const commands = await searchSource.query(inputText);`?